src/main/webapp/app/grading-system/grading-system.component.html

```diff
@@ -57,21 +79,45 @@ <h2 style="margin-bottom: 0.5em">
                 </tr>
             </thead>
             <tbody>
-                <tr *ngFor="let gradeStep of gradingScale.gradeSteps; let i = index">
+                <tr *ngFor="let gradeStep of gradingScale.gradeSteps; let i = index" class="grading-scale-table-row">
+                    <td>
+                        <input [(ngModel)]="gradeStep.lowerBoundPercentage" type="number" step="0.01" min="0" max="100" required (change)="setPoints(gradeStep, true)" />
+                    </td>
+                    <td>
+                        <input
+                            [(ngModel)]="gradeStep.lowerBoundPoints"
+                            type="number"
+                            step="0.01"
+                            min="0"
+                            [max]="getMaxPoints() || 999"
+                            [required]="maxPointsValid()"
+                            [disabled]="!maxPointsValid()"
+                            (change)="setPercentage(gradeStep, true)"
+                        />
+                    </td>
                     <td>
-                        <input [(ngModel)]="gradeStep.lowerBoundPercentage" type="number" min="0" max="100" required />
+                        <input [(ngModel)]="gradeStep.upperBoundPercentage" type="number" step="0.01" min="0" max="100" required (change)="setPoints(gradeStep, false)" />
                     </td>
                     <td>
-                        <input [(ngModel)]="gradeStep.upperBoundPercentage" type="number" min="0" max="100" required />
+                        <input
+                            [(ngModel)]="gradeStep.upperBoundPoints"
+                            type="number"
+                            step="0.01"
```

Comment:
0.01 should only be used for rounding. I don't think using it as step value makes a lot of sense